src/pages/ContentScripts/features/repo-sidebar-labels/index.tsx

```diff
@@ -1,57 +1,58 @@
 import features from '../../../../feature-manager';
-import View from './view';
-import elementReady from 'element-ready';
 import { getRepoName, hasRepoContainerHeader, isPublicRepoWithMeta } from '../../../../helpers/get-repo-info';
-import { RepoMeta, metaStore } from '../../../../api/common';
+import { CommonMeta, RepoMeta, metaStore } from '../../../../api/common';
+import { createRoot } from 'react-dom/client';
+import OpenDiggerLabel from './OpenDiggerLabel';
+
 import React from 'react';
 import $ from 'jquery';
-import { createRoot } from 'react-dom/client';
 
 const featureId = features.getFeatureID(import.meta.url);
-type Label = {
-  id: string;
-  name: string;
-  type: string;
-};
 
 let repoName: string;
 let meta: RepoMeta;
-let filteredLabels: Label[];
+let filteredLabels: CommonMeta['labels'];
 
 const getData = async () => {
   meta = (await metaStore.get(repoName)) as RepoMeta;
   // filtered all xxx-n and n is not 0
-  filteredLabels = (meta.labels as Label[]).filter((label: Label) => {
+  filteredLabels = meta.labels.filter((label) => {
```

Comment:
Add a judgment on meta.labels. Without data from meta.labels, an error will be reported. For example https://github.com/langchain-ai/chat-langchain .
<img width="822" alt="image" src="https://github.com/user-attachments/assets/f0a25e79-edcb-4783-8255-ffeca94141a7">

Reply:
@wangyantong2000 Thanks for reminding :) fixed.